pageserver/src/tenant.rs

```diff
@@ -7898,11 +7910,34 @@ mod tests {
         verify_result().await;
 
         let cancel = CancellationToken::new();
-        tline.compact_with_gc(&cancel, &ctx).await.unwrap();
+
+        tline
+            .compact_with_gc(
+                &cancel,
+                {
+                    let mut flags = EnumSet::new();
+                    flags.insert(CompactFlags::DryRun);
+                    flags
```

Comment:
I think it's better to have this declared outside like:

```Rust
let cancel = ...
let flags = ...
flags.insert(...);
timeline.compact_with_gc(...)
```

Reply:
resolved in https://github.com/neondatabase/neon/pull/8557/commits/f535877ab40fe5ceb1ca856cd86a85acb5901dbb